CONTRIBUTING.md

```diff
@@ -21,6 +21,18 @@ Due to legal reasons, all contributors must sign a contributor agreement, either
 * Tag PRs with either `toward: #<bug>` or `fixes: #<bug>` to help establish context for why a change is happening
 * Commit messages follow [Chris Beam's awesome commit message style guide](http://chris.beams.io/posts/git-commit/)
 
+### Go error reporting
+
+* For problems that are conceptually `asserts`, meaning that the program cannot meaningfully continue, use the following calls:
+	* d.Chk.Error()
+	* d.Chk.NoError()
+	* d.Chk.True()
+	* d.Chk.False()
```

Comment:
I'm not sure if you're intent is to limit people to whqt you have listed here, but I find d.Chk.Nil() and d.Chk.NotNil() to be pretty useful.